Content.Client/GameObjects/Components/Disposal/DisposalUnitWindow.cs

```diff
@@ -33,28 +34,26 @@ public DisposalUnitWindow()
                     {
                         Children =
                         {
-                            new Label {Text = Loc.GetString("State: ")},
-                            (_unitState = new Label {Text = Loc.GetString("Ready")})
+                            new Label {Text = Loc.GetString("ui-disposal-unit-label-state")},
+                            new Control {MinSize = (4, 0)},
```

Comment:
Can probably replace this with `HBoxContainer.SeparationOverride = 4` on the above HBoxContainer.